app/controllers/s3_multipart/uploads_controller.rb

```diff
@@ -3,7 +3,7 @@ class UploadsController < ApplicationController
 
     def create
       begin
-        upload = Upload.create(params)
+        upload = Upload.create(upload_params)
```

Comment:
Might be more readable to do the casting here?

Reply:
Yeah I originally did this but had to update everywhere. Maybe i could name it better?

Reply:
Oh yeah didn't even see it in the other methods. NBD then